scss/_patterns_tooltips.scss

```diff
@@ -53,6 +56,17 @@ $triangle-height: $sp-unit;
       position: absolute;
       width: 0;
     }
+
+    // hover overlay
+    &::after {
+      border-radius: 10% 90% 0% 100% / $bottom-border-radius;
+      content: '';
+      height: #{2 * $triangle-height};
```

Comment:
Isn't `height: 2 * $triange-height;` working here? `#{}` should not be needed on such simple calculations.
Should also be true with `-2 * $triangle-height`.

Reply:
You're right - I had used the same format I saw elsewhere in this file, but on testing it makes no difference, so have removed it from the example I used too.